### DISCUSSION TITLE
Code Push feature

### DISCUSSION THREAD

**Discussion by VincentHoong** [capacitor-community/appcenter-sdk-capacitor#13] [Ideas]
This will be a good plugins as it can take as an alternative for AppFlow.
Looking forward for Code Push feature~

**Comment by johnborges** [comment]
@VincentHoong I left CodePush, out of this repo. While CodePush is part of the AppCenter offering, it is large enough in scope and separate from the core App Center sdk (it even uses different API keys). In staying aligned with the official AppCenter SDKs, I decided to leave it out.

There are some folks who are working on a separate plugin for CodePush. Follow this [proposal](https://github.com/capacitor-community/proposals/issues/43) for more info. It seems someone already has a [working version](https://github.com/mapiacompany/capacitor-codepush) in progress too.
